src/Configurator.php

```diff
@@ -8,6 +8,12 @@
 
 final class Configurator
 {
+    const CONFIG_PREFIX                     = 'config_prefix';
+    const CONFIG_SEPARATOR                  = 'config_separator';
+    const CONFIG_SERVICES_KEY               = 'services_key';
+    const CONFIG_INFLECTORS_KEY             = 'inflectors_key';
+    const CONFIG_DEFAULT_SINGLETON_SERVICES = 'default_singleton_services';
```

Comment:
@tomphp 

I hope you don't already regret inviting me to be a collaborator on this, but how about (for consistency), using `SETTING_` as prefix?

```php
const SETTING_CONFIG_PREFIX              = 'config_prefix';
const SETTING_CONFIG_SEPARATOR           = 'config_separator';
const SETTING_SERVICES_KEY               = 'services_key';
const SETTING_INFLECTORS_KEY             = 'inflectors_key';
const SETTING_DEFAULT_SINGLETON_SERVICES = 'default_singleton_services';
```

It's quite long, but maybe it's less confusing, what do you think?